### ISSUE TITLE
Бесконечная лента

### ISSUE THREAD

**Issue by solarrust** [doka-guide/content#2842]
Реализация бесконечной ленты постов (или других итемов) на чистом JS с использованием Intersection Observer API

**Comment by nlopin** [comment]
У нас уже есть такая [статья в разделе JS](https://doka.guide/js/infinite-scroll/)! Думаю можно её перечитать и превратить в рецепт

**Comment by solarrust** [comment]
Да, из неё получился бы отличный рецепт 👍
Можно даже пофетчит реальные данные откуда-нибудь типа https://jsonplaceholder.typicode.com

**Comment by nikitadubyk** [comment]
Привет! Мне было бы интересно сделать такую статью. Можно взяться за ее реализацию? Если что, можно брать материал с уже существующей статьи?

**Comment by nlopin** [comment]
Да, можно брать и пролинковывать даже. В качестве апи можно ещё воспользоваться rickandmortyapi.com

**Comment by solarrust** [comment]
Привет! Как дела с этой задачкой?
